docs/GettingStarted.md

```diff
@@ -132,6 +132,28 @@ Before downloading tracing on your application, install the Datadog Agent. The R
 
 ### Quickstart for Rails applications
 
+#### Rails Auto Instrument all Integrations
+
+1. Add the `ddtrace` gem to your Gemfile:
+
+    ```ruby
+    source 'https://rubygems.org'
+    gem 'ddtrace'
+    ```
+
+2. Install the gem with `bundle install`
+
+3. Add `require 'ddtrace/auto_instrument'` to your application startup sequence. This can be done in your `config/application.rb` file.
+
+    ```ruby
+      require 'rails/all'
+      require 'ddtrace/auto_instrument'
```

Comment:
Remove indentation level, as these require statements will be pasted at the top-level of their project file.
```suggestion
    require 'rails/all'
    require 'ddtrace/auto_instrument'
```